src/gui/newwizard/jobs/checkbasicauthjobfactory.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+* Copyright (C) Fabian Müller <fmueller@owncloud.com>
+*
+* This program is free software; you can redistribute it and/or modify
+* it under the terms of the GNU General Public License as published by
+* the Free Software Foundation; either version 2 of the License, or
+* (at your option) any later version.
+*
+* This program is distributed in the hope that it will be useful, but
+* WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+* or FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License
+* for more details.
+*/
+
+#include "checkbasicauthjobfactory.h"
+
+#include "accessmanager.h"
+#include "common/utility.h"
+#include "creds/httpcredentials.h"
+#include "gui/tlserrordialog.h"
+#include "gui/updateurldialog.h"
+#include "theme.h"
+
+#include <QNetworkReply>
+
+namespace OCC::Wizard::Jobs {
+
+CoreJob *CheckBasicAuthJobFactory::startJob(const QUrl &url)
+{
+    auto *job = new CoreJob;
+
+    QNetworkRequest req(Utility::concatUrlPath(url, Theme::instance()->webDavPath()));
+
+    req.setAttribute(HttpCredentials::DontAddCredentialsAttribute, true);
```

Comment:
Not needed as we don't use a credential manager yet (HttpCredentialsAccessManager)